src/components/tree/infoPanels/hover.js

```diff
@@ -267,14 +262,13 @@ const BranchMutations = ({node, geneSortFn, observedMutations, t}) => {
  * @param  {Object} props
  * @param  {Object} props.node  branch node which is currently highlighted
  */
-const BranchDescendents = ({node, t}) => {
+const BranchDescendants = ({node, t, tipLabelKey}) => {
   const [name, value] = node.fullTipCount === 1 ?
-    [t("Branch leading to"), node.name] :
+    [nodeDisplayName(node, tipLabelKey, true), ""] :
     [t("Number of descendants")+":", node.fullTipCount];
```

Comment:
This is displaying `undefined`:

![image](https://github.com/user-attachments/assets/475d67b5-aece-46c1-b9d5-5b17501ad8e2)

Fix is simple:

```suggestion
    [nodeDisplayName(t, node, tipLabelKey, true), ""] :
    [t("Number of descendants")+":", node.fullTipCount];
```

I was confused how this could happen without any warnings or errors, and realized that [it's by design](https://stackoverflow.com/q/11107823) 😵 I'm guessing that's why it's common practice to pass multiple values as a single object with key/value pairs.

Reply:
> why it's common practice to pass multiple values as a single object with key/value pairs

That's a useful approach when you want to have multiple optional values (positional & optional becomes painful unless it's the last parameter).

IMO the fix here is to leverage TypeScript rather than changing function signatures

Thanks for identifying the bug tho! Fixed [in force push](https://github.com/nextstrain/auspice/compare/962a2e24e46a5a6f9b28a9157deebd662d9d08db..399c50181ab1ed35a88c1a6abc95b91193502eb0)